flexmeasures_openweathermap/utils/owm.py

```diff
@@ -211,3 +213,11 @@ def save_forecasts_as_json(
         )
         with open(forecasts_file, "w") as outfile:
             json.dump(forecasts, outfile)
+
+
+def check_openweathermap_version(api_version: str):
+    supported_versions = ["2.5", "3.0"]
+    if api_version not in supported_versions:
+        raise Warning(
```

Comment:
I'd rather log a warning than `raise` (which I'm assuming stops everything).

Reply:
`current_app.logger.warning()` would just send a warning, and the script would continue to run and try to get the data. Consider I pass `2.0` version, the function sent a warning but the api is still called and failed because of the incompatible version. 

![image](https://github.com/SeitaBV/flexmeasures-openweathermap/assets/59763365/77562f70-e7ee-488a-aa15-a4def65eefcf)

Reply:
Well, we don't know which versions *would* work. And imagine they come out with version 3.1 in a few weeks, and then fm-owm stops working, even though it would very likely work fine if we didn't raise.
Se should only work on what we know for sure.
If it fails for using version 2, they can see why in the logs.